presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -200,7 +208,41 @@ public static void main(String[] args)
     {
         // You need to add "--user user" to your CLI for your queries to work
         Logging.initialize();
-        DistributedQueryRunner queryRunner = createQueryRunner(TpchTable.getTables(), ImmutableMap.of("http-server.http.port", "8080"));
+
+        Optional<Path> baseDirPath = Optional.empty();
+        if (args.length > 0) {
+            if (args.length != 1) {
+                System.err.println("usage: HiveQueryRunner [baseDir]\n");
```

Comment:
Please rename `baseDir`, it's not easy to understand what it means. Also, we use that in the error messages below, we should update those to be meaningful too.

Reply:
Changed to the following:
```
log.error("usage: HiveQueryRunner [baseDataDir]\n");
                log.error("       [baseDataDir] is a local directory that you want the hive_data directory will be created under.]\n");
```